roles/openshift_master_facts/tasks/main.yml

```diff
@@ -30,10 +30,10 @@
       embedded_etcd: "{{ openshift_master_embedded_etcd | default(None) }}"
       embedded_kube: "{{ openshift_master_embedded_kube | default(None) }}"
       embedded_dns: "{{ openshift_master_embedded_dns | default(None) }}"
-      dns_port: "{{ openshift_master_dns_port | default(None) }}"
+      # defaults to 8053 when using dnsmasq in 1.2/3.2
+      dns_port: "{{ openshift_master_dns_port | default('53' if not openshift.common.use_dnsmasq else None ) }}"
```

Comment:
See my comment on the openshift_facts module, I'd rather see the default handled there. openshift_common is pulled in prior to openshift_master_facts being run (through openshift_cli).